examples/vault-consul-enterprise-image/README.md

```diff
@@ -0,0 +1,11 @@
+# Vault and Consul Enterprise Google Image
+
+## Encrypting your Vault Enterprise license
+
+We recommend encrypting your license key locally using the Google KMS tools.
+
+You can simply extend the Packer template to decrypt your license key when baking the image or during startup in the `start-script-vault.sh` script.
+
+```bash
+$ gcloud kms decrypt --keyring=gruntwork-test --key=vault-test --location=global --ciphertext-file=/opt/vault/vault-license.hclic.enc --plaintext-file=/opt/vault/vault-license.hclic
+```
```

Comment:
Nice, but per my comments in Slack, I'd prefer decrypting the file at boot versus baking a Compute Image with a plaintext license file. This way, we can also avoid building a whole new AMI just because we wanted to change the license file. That being said, perhaps it makes sense to include an encrypted license file in git in the appropriate example folder. You can pass in its content as a Terraform var using the Terraform `file()` interplation as you're populating the Startup Script.

Reply:
@josh-padnick okay, but the Gruntwork encrypted license file in git or just a dummy one? We'd have to setup the Gruntwork GCP account and properly configure a KMS key for use with the automated tests. I'd encrypt the license key using the Gruntwork GCP key.